package.json

```diff
@@ -35,22 +35,22 @@
     "@babel/plugin-transform-modules-commonjs": "^7.12.1",
     "@glennsl/bs-jest": "^0.6.0",
     "@reasonml-community/graphql-ppx": "^1.0.1",
+    "@rescript/react": "^0.10.1",
     "all-contributors-cli": "^6.19.0",
     "babel-jest": "^26.6.3",
-    "bs-platform": "^8.3.2",
-    "gentype": "^3.38.0",
+    "bs-platform": "^9.0.1",
+    "gentype": "^3.44.0",
     "graphql": "^15.4.0",
     "npm-run-all": "^4.1.5",
     "react": "^16.8.0",
-    "reason-react": "^0.9.1",
-    "urql": "~1.11.2"
+    "urql": "~2.0.1"
```

Comment:
I'd love for this to switch over to a simple minor-range, since future changes shouldn't affect the bindings. I think the patch range was rather temporary, if I'm not mistaken, to prevent other changes from interferring with the bindings slowly catching up, so since we're all caught up now it should be safe to bump this to the minor range

Reply:
Yep, you're right! I agree that a more permissive minor range sounds appropriate!